docker/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# Pulling base image
+FROM ubuntu:18.04
+MAINTAINER Arie Kurniawan "contact.arkwrn@gmail.com"
```

Comment:
I prefer not including any credits in-file. As a matter of recognition I'll give you credits on AUTHORS file.

Reply:
Agree with You, sorry i'll delete it